classes/Divisions.php

```diff
@@ -299,6 +305,23 @@ public function getMemberDivisionDetails($strong_only = false) {
           $policy_divisions[$policy_id] = $summary;
         }
 
+        // for each key in $policy_divisions, we want to add agreement information
+        foreach ($policy_divisions as $policy_id => $summary) {
+            $agreement_details = $this->member->member_agreements($policy_id);
+            $summary["agreements_for"] = 0;
+            $summary["agreements_against"] = 0;
+            foreach ($agreement_details as $agreement){
+                if ($strong_only == true & $agreement["strength"] != "strong") {
+                    continue;
+                }
+                if ($agreement["alignment"] == "agree") {
+                    $summary["agreements_for"] += 1;
+                } else {
+                    $summary["agreements_against"] += 1;
+                }
+            }
+            $policy_divisions[$policy_id] = $summary;
```

Comment:
I don't think this line is needed, because you're altering the $summary object directly above? Might need checking.

Reply:
Yes, if $summary is adjusted to being passed by ref.